src/uct/ib/rc/base/rc_iface.c

```diff
@@ -593,7 +593,7 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_iface_ops_t *tl_ops,
 #endif
     max_ib_msg_size             = uct_ib_iface_port_attr(&self->super)->max_msg_sz;
 
-    if (dev->flags & UCT_IB_DEVICE_FLAG_ECE) {
+    if (ucs_derived_of(md, uct_ib_md_t)->ece_enable) {
```

Comment:
-rename md to tl_md
-add "uct_ib_md_t *md" variable
-use it in line 562 too

Reply:
@yosefe 
For ```uct_md_h md```, it's better to keep the function parameter name as other similar function.
```
UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_iface_ops_t *tl_ops,
                    uct_rc_iface_ops_t *ops, uct_md_h md, uct_worker_h worker,
```
Do you mean to
1) use ```uct_md_h tl_md``` name in the function parameter?
OR
2) keep ```uct_md_h md``` function parameter and define an new local variable ```uct_ib_md_t *tl_md```?

Reply:
option 1

Reply:
Done